composer.json

```diff
@@ -8,41 +8,46 @@
   "repositories": [
     {
       "type": "composer",
-      "url": "https://packagist.drupal-composer.org"
+      "url": "https://packages.drupal.org/8"
     }
   ],
   "require-dev": {
+    "behat/behat": "^3.0",
     "behat/mink": "~1.7",
     "behat/mink-zombie-driver": "~1.2",
     "behat/mink-goutte-driver": "~1.2",
-    "drupal/drupal-extension": "~3.0",
+    "drupal/drupal-extension": "~3.2",
     "jcalderonzumba/gastonjs": "~1.0.2",
     "jcalderonzumba/mink-phantomjs-driver": "~0.3.1",
     "mikey179/vfsStream": "~1.2",
+    "se/selenium-server-standalone": "^2.53",
     "phpunit/phpunit": "~4.8",
     "drush/drush": "^8",
     "drupal/console": "~1.0.0",
     "phpmd/phpmd": "~2.1",
-    "drupal/admin_toolbar": "~8.1",
+    "drupal/admin_toolbar": "^1.18",
     "drupal/coder": "^8.2",
```

Comment:
Doesn't drupal/coder also need a new version for the d.o packagist? Like ^2.0 or something.